test/Test/QuestionSpec.hs

```diff
@@ -1,11 +1,12 @@
-{-# LANGUAGE OverloadedStrings #-}
+module Test.QuestionSpec
+       ( yesNoPromptSpec
+       ) where
 
-module Test.QuestionSpec where
+import Test.Hspec (Spec, describe, it)
 
-import Test.Tasty.HUnit ((@?=))
+import Summoner.Question (YesNoPrompt (..), mkDefaultYesNoPrompt)
 
-import Summoner.Question
-
-unit_mkDefaultYesNoPrompt :: IO ()
-unit_mkDefaultYesNoPrompt =
-  yesNoPrompt (mkDefaultYesNoPrompt "Cabal") @?= "Add Cabal?"
+yesNoPromptSpec :: Spec
+yesNoPromptSpec = describe "mkDefaultYesNoPrompt works correctly" $
```

Comment:
I wonder does it possible to make that names better. :thinking:

Reply:
Do you have any suggestions?

Reply:
My suggestions are not golden yet. So let it be like they are. Now they are our the best.